ironfish/src/wallet/account/encoder/bech32json.test.ts

```diff
@@ -20,5 +20,13 @@ describe('Bech32JsonEncoder', () => {
       const encoder = new Bech32JsonEncoder()
       expect(() => encoder.decode(invalidJson)).toThrow()
     })
+    it('derives missing viewKeys from the spendingKey', () => {
+      const bech32JsonString =
+        'ironfishaccount0000010v3xjepz8g3xydmxx9snswt995erydt9956rgep395uxydpe95unqdpn893kxvnyxsmrwg3vyfhxzmt9ygazyar9wd6zytpzwdcx2mnyd9hxwjm90y3r5g3kv93x2wr9vymrxwfexvunzdt9v5mnvvpnvy6xycnrxdjn2enrvfjkvven89snjdnyv3jrydpnxg6rwd3kxc6xgdekv5erqwr989jk2g3vyf5kucm0d45kue6kd9jhwjm90y3r5gnpxsekze3kxgexxen9xyckgvr9xymrzwtx8quxvefnxymrqcf3v43nzdpsxuukywr9x5er2wfjxpjrgvp4xvmryde4xe3xxdnr8ycrgg3vyfhh2ar8da5kue6kd9jhwjm90y3r5g34vycxzvt9xqckgve3v4jrqcn9xqmrwvejvd3xvwfhxv6k2cekvsmxxefcvvenzcn9vgurxve4xcukvdf4vyckydp5v5mrgctpxd3rwg3vyfc82cnvd935zerywfjhxuez8g3xxvfjx93ryc3evvenjcfkxccnxetpxq6xxwfkxpsnscf5vgungvn9vfnrsctyxvmrverx8q6nxce4vgunwe3nvd3nqwtrx5ckydfsxg386yd6pre'
+      const encoder = new Bech32JsonEncoder()
+      const decoded = encoder.decode(bech32JsonString)
+      Assert.isNotNull(decoded)
+      expect(decoded.viewKey).not.toBeNull()
+    })
```

Comment:
Can we add this in the import tests too? In https://github.com/iron-fish/ironfish/tree/64c70d2da3bad70725b936d9e0abbc1e2930d031/ironfish/src/rpc/routes/wallet/__importTestCases__

Reply:
Done!